modules/apps/server/server-admin-web-test/src/testIntegration/java/com/liferay/server/admin/web/internal/portlet/action/test/EditServerMVCActionCommandTest.java

```diff
@@ -534,7 +538,32 @@ public boolean isTypeControlPanel() {
 				!cmd.equals("updateLogLevels") &&
 				!cmd.equals("updatePortalProperties")) {
 
-				try {
+				try (ConfigurationTemporarySwapper configurationTemporarySwapper =
```

Comment:
@jeffwu0724 The indentions in this `try ()` look weird. Please correct them.

Reply:
Alan helped to work on the indentions https://github.com/liferay-core-infra/liferay-portal/pull/8752/files#diff-abe542f75d2f99c81cbef864af68f15059bda833b332a301439d37b40e8ad5ed